src/main/webapp/resources/js/apis/projects/samples.ts

```diff
@@ -129,29 +142,40 @@ export async function getLockedSamples({
   return response.data;
```

Comment:
Can you type this request so the return value is properly typed please. This is something we all need to make sure we are doing.

Reply:
Return type is Promise<LockedSamplesResponse>.